archive-gs-factsheet.php

```diff
@@ -98,13 +114,44 @@
 							<ul>
 						<?php
 					}
+
+					if ( 1 < count( $factsheet ) ) {
+						$wrapper_class = 'degree-row-multiple';
+					} else {
+						$wrapper_class = 'degree-row-single';
+					}
 					?>
-					<li>
-						<div class="degreename flexleft"><a><?php echo esc_html( $factsheet_name ); ?></a></div>
+					<li class="degree-row-wrapper <?php echo esc_attr( $wrapper_class ); ?>">
+						<div class="degree-row-top">
+							<?php
+							if ( 1 < count( $factsheet ) ) {
+								?><div class="degree-name"><span class="degree-anchor"><?php echo esc_html( $factsheet_name ); ?></span><?php
+							} else {
+								?><div class="degree-name"><a href="<?php echo esc_url( $factsheet[0]['permalink'] ); ?>"><?php echo esc_html( $factsheet_name ); ?></a><?php
+							}
+							?>
+							</div>
+							<?php
+							foreach ( $factsheet as $item ) {
+								?>
+								<div class="degree-classification <?php echo esc_attr( $item['degree_classification'] ); ?>">
+									<?php echo esc_html( $item['degree_classification'][0] ) ?>
```

Comment:
This is actually a "clever" way of outputting the first character of the classification for use as a key. It took me a minute to figure this out even though I wrote it! I'll add a comment so that it's more obvious.